sdk/eventhubs/azure-messaging-eventhubs/src/producer_client.cpp

```diff
@@ -54,10 +57,13 @@ void Azure::Messaging::EventHubs::ProducerClient::CreateSender(std::string const
 
   Azure::Core::Amqp::_internal::ConnectionOptions connectOptions;
   connectOptions.ContainerId = m_producerClientOptions.ApplicationID;
-  connectOptions.EnableTrace = m_producerClientOptions.SenderOptions.EnableTrace;
-  std::string hostName;
-  hostName = m_fullyQualifiedNamespace;
+  connectOptions.EnableTrace = m_producerClientOptions.VerboseLogging;
+  connectOptions.AuthenticationScopes = {"https://eventhubs.azure.net/.default"};
```

Comment:
Is this scope correct for sovereign clouds?

Reply:
I have absolutely no idea. However this is the corresponding code from the GO implementation:
```go
	accessToken, err := tpa.tokenCred.GetToken(context.TODO(), policy.TokenRequestOptions{
		Scopes: []string{
			"https://eventhubs.azure.net//.default",
		},
```